### DISCUSSION TITLE
How to create Template for Rowy.

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#856] [Support / Q&A] [answered]
I am working on a tech article that will be complemented with a demo project for Rowy.
Is there any doc I should follow while creating a demo project for Rowy?
How can I hand over the demo project to the core team?
By demo, I mean: https://demo.rowy.io/tables

**Comment by harinij** [comment, marked as answer]
@RajGM currently we do not have a way for submission to the showcase demo apps, but if you want to submit a template for cloud functions you can do so here along with the detailed readme https://github.com/rowyio/templates
